clang/include/clang/Sema/BoundsAnalysis.h

```diff
@@ -266,6 +263,17 @@ namespace clang {
     // @param[in] FD is the current function.
     void CollectNtPtrsInScope(FunctionDecl *FD);
 
+    // Get the Kill set for the current block. The Kill set is a mapping of
+    // Stmts to variables whose bounds killed by each Stmt in the block.
+    // @param[in] B is the current CFGBlock.
+    // return A mapping of Stmts to variables whose bounds killed by the Stmt.
+    StmtDeclSetTy GetKillSet(const clang::CFGBlock *B);
```

Comment:
Is GetKillSet intended to be called from CheckBoundsDeclarations? It looks like it's a private method.